packages/testkit/eslint.config.js

```diff
@@ -0,0 +1,6 @@
+// eslint.config.js
+export default [
+  {
+    ignores: ["node_modules/", "dist/", "*.config.js", "*.spec.js"],
```

Comment:
Why this? It should already ignore node_modules and dist
For the others I think it'd be useful